Makefile

```diff
@@ -89,6 +89,10 @@ prod-docker:
 prod-test:
 	RUSTFLAGS="--cfg disable_faketime" RUSTDOCFLAGS="--cfg disable_faketime" cargo test ${VERBOSE} --all -- --nocapture
 
+.PHONY: prod-with-debug
+prod-with-debug:
+	devtools/release/make-with-debug-symbols prod
```

Comment:
Could you try `make RUSTFLAGS=-g prod`?

Reply:
When use `debug = true`, the size of debug symbols in executable file is bigger than `-g` (or `-C debuginfo=2`), and `debug = true` will enable some other debug features, for example: `debug_assertions = true`.

Reply:
I tried to use `objdump` to figure out why the part of debug symbols is bigger when use `debug = true` yesterday, but I found nothing (I'm not good at it). So I use a script to change the `Cargo.toml` to implement this feature (reduce the size).